tools/online_editor/cypress/plugins/index.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+// cSpell: ignore frsource
+
+import "@frsource/cypress-plugin-visual-regression-diff/dist/support";
+
+module.exports = (on, _config) => {
+  on("before:browser:launch", (browser = {}, args: string[]) => {
+    if (browser.name === "chromium" || browser.name == "chrome") {
+      const newArgs = args.filter((arg) => arg !== "--disable-gpu");
+      newArgs.push("--ignore-gpu-blacklist");
+      return newArgs;
```

Comment:
It appears that cypress deliberately disables GPU based rendering, and this "plugin" attempts to remove that again.

I'm curious, do you know why this is needed?

Reply:
Cypress disabled it because they found chrome on Linux unstable when using GPU acceleration.

I wanted to see whether that actually causes issues in our CI. There is no preview without WebGL after all.

Reply:
I mean, even if the GH action CI machines would expose a GPU under Linux (which AFAIK they don't), it would not be something we would want to use IMO. The driver setup is far out of our control.

What I believe we *want* to use is Chrome's software rasteriser, also for WebGL (Swiftshader). That is available regardless of whether you pass --disable-gpu on the command line or not. To the best of my knowledge this merely allows or disallows Chrome to use hardware acceleration.

You can try this out yourself, btw: Run chrome, go to "about:gpu". You'll see OpenGL disabled, hardware acceleration generally disabled, but WebGL will be shown as using software rendering. That is what we want IMO.

I don't think overriding the cypress settings is correct here.